resources/views/processes/index.blade.php

```diff
@@ -0,0 +1,30 @@
+@extends('layouts.layout', ['title' => __('Processes Management')])
+
+@section('sidebar')
+    @include('layouts.sidebar', ['sidebar'=> Menu::get('sidebar_admin')])
```

Comment:
This is the wrong sidebar, but we don't have this one specifically.

Reply:
we changed to  sidebar_designer